spec.html

```diff
@@ -19955,10 +19955,10 @@ <h1>Static Semantics: BoundNames</h1>
       </emu-alg>
       <emu-grammar>FunctionDeclaration : `function` `(` FormalParameters `)` `{` FunctionBody `}`</emu-grammar>
       <emu-alg>
-        1. Return &laquo; *"\*default\*"* &raquo;.
+        1. Return &laquo; ~default~ &raquo;.
       </emu-alg>
       <emu-note>
-        <p>*"\*default\*"* is used within this specification as a synthetic name for hoistable anonymous functions that are defined using export declarations.</p>
+        <p>~default~ is used within this specification as a synthetic name for hoistable anonymous functions that are defined using export declarations.</p>
```

Comment:
It may be a bit outside the scope of this PR, but I'd love to see a sentence like this added where appropriate.

```suggestion
        <p>~default~ is used within this specification as a synthetic name for hoistable anonymous functions that are defined using export declarations. It is never accessible from user code.</p>
```

Reply:
Seems straightforward and I'm on board; I suppose I could add it in another commit here if you prefer.

Reply:
This occurs 3 times in the spec; do you want them all updated?

Reply:
Yeah let's do it.